backend/models/cans.py

```diff
@@ -770,10 +769,24 @@ def status(self):
         )
         return can_status
 
+    @property
+    def appropriation_term(self):
+        if self.expiration_date is None or self.appropriation_date is None:
```

Comment:
It may be slightly better to return `None` when `self.appropriation_date is None` - `0` year money means the funding doesn't expire and so has a different meaning from if `appropriation_date` is `NULL`/unknown

Reply:
I'll make this change.